File: lib/app/Models/OrdersStatsDetailsDTO.php
<?php
/**
 * OrdersStatsDetailsDTO
 */
namespace app\Models;

/**
 * OrdersStatsDetailsDTO
 * @description Информация об удалении товара из заказа.
 */
class OrdersStatsDetailsDTO {

    /** @var string $itemStatus */
    public $itemStatus = "";

    /** @var int $itemCount Количество товара со статусом, указанном в параметре &#x60;itemStatus&#x60;.*/
    public $itemCount = 0;

    /** @var \DateTime $updateDate Дата, когда товар получил статус, указанный в параметре &#x60;itemStatus&#x60;.  Формат даты: &#x60;ГГГГ-ММ-ДД&#x60;.*/
    public $updateDate;

    /** @var string $stockType */
    public $stockType = "";

}

File: ympa_fsharp_functions_server/src/model/OrdersStatsDetailsDTO.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.OrdersStatsItemStatusType
open ympa_fsharp_functions_server.Model.OrdersStatsStockType

module OrdersStatsDetailsDTO =

  //#region OrdersStatsDetailsDTO

  [<CLIMutable>]
  type OrdersStatsDetailsDTO = {
    [<JsonProperty(PropertyName = "itemStatus")>]
    ItemStatus : OrdersStatsItemStatusType;
    [<JsonProperty(PropertyName = "itemCount")>]
    ItemCount : int64;
    [<JsonProperty(PropertyName = "updateDate")>]
    UpdateDate : DateTime;
    [<JsonProperty(PropertyName = "stockType")>]
    StockType : OrdersStatsStockType;
  }

  //#endregion
